src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/UpToDateCheckImplicitConfiguredInput.cs

```diff
@@ -678,12 +678,11 @@ ProjectCopyData UpdateCopyData()
                     jointRuleUpdate.ProjectChanges.TryGetValue(ResolvedProjectReference.SchemaName, out IProjectChangeDescription? change2) &&
                     jointRuleUpdate.ProjectChanges.TryGetValue(ConfigurationGeneral.SchemaName, out IProjectChangeDescription? change3))
                 {
-                    if (change1.Difference.AnyChanges || change2.Difference.AnyChanges || change3.Difference.AnyChanges)
+                    if ((change1.Difference.AnyChanges || change2.Difference.AnyChanges || change3.Difference.AnyChanges) &&
+                        change3.After.Properties.TryGetStringProperty(ConfigurationGeneral.TargetPathProperty, out string? targetPath))
```

Comment:
what's the difference here on getting the property from the `.ProjectChanges`? I noticed before we were getting it from the `CurrentState`.

Reply:
The delta contains a pointer to the previous and updated state. In this case the after state is the same as the current state, and I felt that writing it this way made the code clearer.